appserver/admingui/SNMP-notifier-console-plugin/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>fish.payara.admingui</groupId>
+        <artifactId>admingui</artifactId>
+        <version>4.1.1.171-SNAPSHOT</version>
+    </parent>
+    <artifactId>SNMP-notifier-console-plugin</artifactId>
```

Comment:
Can we make this and the respective folder lowercase please? It's convention for them to be all lowercase